### ISSUE TITLE
Feature Request: Common Test Dataset

### ISSUE THREAD

**Issue by Btibert3** [neo4j-rstats/neo4r#54]
Not a bug, but as I am looking at the docs, README, and the [guide](https://neo4j-rstats.github.io/user-guide/connect.html), I think it might be easier if we all had a common dataset to work through as we "learn" the api of `neo4r`.  

It could be neat/easy to do something like `neo4j_api$create_example_graph()`.  I am sure the command could be more succinct, but you get the idea.

This might help with tests and debugging of issues.

**Comment by ColinFay** [comment]
Hey, 

The latest version introduces `play_movies()`, which is a cypher queries to create the movies dataset. 

Current implementation allows to:

``` r 
play_movies() %>%
  call_neo4j(con)
```

Can you think of other Neo4J examples that would be useful?

**Comment by Btibert3** [comment]
No, this would work.   Just something that is standard to help work through examples and highlight new features when they are added.  Worst case, now that this is setup, I could always create a PR if I think of something else.  Thanks!

**Comment by ColinFay** [comment]
Perfect :) 

I've adjusted the README and the User guide, and now they are based on the movie dataset so it's more familiar for people coming from Neo4J :)
